dev-python/mongoengine/mongoengine-0.25.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python Object-Document-Mapper for working with MongoDB"
+HOMEPAGE="
+	https://github.com/MongoEngine/mongoengine
+	http://mongoengine.org/
+"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	dev-db/mongodb
```

Comment:
I do wonder if MongoDB is really a RDEPEND here. FWICS in pymongo we treat it purely as test dep, I don't think you technically need to have the server installed locally to use the Python package.

Reply:
Yes. I would wonder too. If I'd remove it from ebuild, and it were actually required, would some CI run the test and catch test are broken? Hence my broader question in the checklist:
> Verify dependencies are actually satisfied on a clean system. Does some QA pipeline check this automatically or needs to be checked manually?